frontend/icons/src/main/java/icons/MacheteIcons.java

```diff
@@ -13,35 +13,35 @@ private MacheteIcons() {}
 
   public static final Icon DISCOVER = loadIcon("/macheteLogoIcon.svg");
 
+  public static final Icon EDIT = loadIcon("/edit.svg");
+
   public static final Icon FAST_FORWARD = loadIcon("/applyNotConflictsLeft.svg");
 
-  public static final Icon FETCH = loadIcon("/download.svg");
+  public static final Icon FETCH = loadIcon("/fetch.svg");
 
   public static final Icon HELP = loadIcon("/help.svg");
 
   public static final Icon MACHETE_FILE = loadIcon("/macheteLogoIcon.svg");
 
-  public static final Icon MACHETE_LOAD_CHANGES = loadIcon("/macheteLoadChanges.svg");
+  public static final Icon MERGE_PARENT = loadIcon("/merge.svg");
```

Comment:
Huh actually... amirite that if a typo occurs in the icon name/path... an exception is NOT thrown for some reason? 🤔 instead, just an empty placeholder shows

Pls verify, if so, pls open an issue for some extra safety here (so that an exception is actually thrown)

Reply:
https://github.com/VirtusLab/git-machete-intellij-plugin/issues/842